app/templates/components/event-table.hbs

```diff
@@ -0,0 +1,58 @@
+<div class = "ui grid container">
+  <div class = "row">
+    <div class = "sixteen wide column">
+      <table class="ui compact  celled table">
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Date</th>
+            <th>Roles</th>
+            <th>Sessions</th>
+            <th>Speakers</th>
+            <th>Tickets</th>
+            <th>Public Url</th>
+            <th>Actions</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <div><i class="large camera icon"></i></div>
```

Comment:
Don't shown icon please. Show a placeholder image with services like https://placehold.it/.